public/demo.js

```diff
@@ -173,8 +173,8 @@ $(document).ready(function() {
             return false;
         }
 
-        if (($("#dropdownMenuInput").text()).toLowerCase().indexOf("language") < 0) {
-            var source = getLanguageCode($.trim($("#dropdownMenuInput").text()));
+        if (getSrcLang().toLowerCase().indexOf("language") < 0) {
```

Comment:
I don't think it would return Choose or Detect. Since the label of that menu item would be overwritten by Language Identification output, where the first word is going to be the language always.

There could be an edge case somewhere here when the label is not yet updated.